lib/MockXMLHttpRequest.js

```diff
@@ -171,6 +165,8 @@ MockXMLHttpRequest.prototype.send = function(data) {
         }, typeof(timeout) === 'number' ? timeout : self.timeout+1);
 
       } else {
+        //trigger a load event to indicate the data has been sent
+        self.upload.trigger('load');
```

Comment:
`onloadstart` too? https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/upload

we should probably dispatch the other events e.g. `onloadend` at some stage too?